Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12.1-alpine3.9 AS builder
+
+COPY . /go/src/github.com/Huawei-PaaS/ci-bot
```

Comment:
The path `/go/src/github.com/` seems does not work for everyone.

Reply:
COPY command copies all the source files of your host directory to respective container directory...
![Screenshot from 2019-04-05 09-30-20](https://user-images.githubusercontent.com/39593583/55602904-8bd88300-5785-11e9-9662-82497d57b079.png)
It will work fine i think
![Screenshot from 2019-04-05 09-50-58](https://user-images.githubusercontent.com/39593583/55604398-3b652380-578d-11e9-889b-1466ab22b88f.png)

Reply:
oh u mean i must make it generic so that all projects
ex like cni genie, kube volcano ..if they do docker build-image it must build their image (ex kube-volcano) ????    Is that what you need???

Reply:
I mean probably different people may have different go path.

Reply:
I think the base image golang:1.12.1-alpine3.9 has GOPATH set to /go. Since the second argument is the target(which is image in our case), I dont think it will be a problem.,

Reply:
Okay, thanks for clarification. If so, I can buy it.